### PR TITLE
[DS-3615] Allows changing of value-pairs-name for language tag.

### PR DESCRIPTION
**Pull request by edusperoni**
https://jira.duraspace.org/browse/DS-3615
If this gets approved I'll port it to master.

### CODE REVIEW

File: dspace-api/src/main/java/org/dspace/app/util/DCInput.java [hunk 1]
```diff
@@ -114,7 +114,8 @@ public DCInput(Map<String, String> fieldMap, Map<String, List<String>> listMap)
         valueLanguageList = new ArrayList();
         if (language)
         {
-            valueLanguageList = listMap.get(LanguageName);
+            languageName = fieldMap.get("language-value-pairs-name");
```
Comment (AlexanderS) [inline dspace-api/src/main/java/org/dspace/app/util/DCInput.java]: Can we use the default value here, if no language-value-pairs-name is given?

Comment (AlexanderS) [inline dspace-api/src/main/java/org/dspace/app/util/DCInput.java]: Ah scratch this. I just saw that `DCInputsReader` is enforcing the language attribute.

File: dspace-api/src/main/java/org/dspace/app/util/DCInput.java [hunk 2]
```diff
@@ -33,7 +33,7 @@
     private boolean language = false;
     
     /** the language code use for the input */
-    private static final String LanguageName = "common_iso_languages";
+    private String languageName = "common_iso_languages";
```
Comment (AlexanderS) [inline dspace-api/src/main/java/org/dspace/app/util/DCInput.java]: This can be `null`. No need for a value, that is never used.

Comment (edusperoni) [inline dspace-api/src/main/java/org/dspace/app/util/DCInput.java]: Good call, added it to the PR.

**Review by AlexanderS — APPROVED** [review]
Tested it and it works.

**Pull request closed without merging**